### PR TITLE
Release 1.1.6

### PR DESCRIPTION
**Pull request by makosblade**
Release 1.1.6
- added write mode 
- updated kotlin

### CODE REVIEW

**Review by pckeyes — APPROVED** [review]

**Review by you-leee — APPROVED** [review]

File: src/main/kotlin/com/openlattice/launchpad/configuration/IntegrationTables.kt [hunk 1]
```diff
@@ -20,7 +21,9 @@ class IntegrationTables {
          * 3. table_name
          * 4. start
          */
-        const val LOG_INTEGRATION_STARTED  = "INSERT INTO integration_activity (integration_name, host_name, table_name, start) VALUES (?,?,?,?)"
+        const val LOG_INTEGRATION_STARTED  = "INSERT INTO integration_activity (integration_name, host_name, table_name, start) " +
```
Comment (dbt) [inline src/main/kotlin/com/openlattice/launchpad/configuration/IntegrationTables.kt:24]: would kotlin multiline strings be better here?
```
val ... = """
    INSERT INTO ...
    VALUES (?, ?, ?, ?)
"""
```

File: src/main/kotlin/com/openlattice/launchpad/configuration/IntegrationTables.kt [hunk 2]
```diff
@@ -29,7 +32,12 @@ class IntegrationTables {
          * 3. table_name
          * 4. start
          */
-        const val LOG_SUCCESSFUL_INTEGRATION = "UPDATE integration_activity SET finish = now(), result = 'SUCCESS' WHERE integration_name = ? AND host_name = ? AND table_name = ? AND start = ? "
+        const val LOG_SUCCESSFUL_INTEGRATION = "UPDATE integration_activity " +
+                "SET finish = now(), result = 'SUCCESS' " +
```
Comment (dbt) [inline src/main/kotlin/com/openlattice/launchpad/configuration/IntegrationTables.kt:36]: same thought as above.

**Review by dbt — APPROVED** [review]
one question, otherwise lgtm.

File: src/main/kotlin/com/openlattice/launchpad/configuration/IntegrationTables.kt (hunk 1 above)
Comment (makosblade) [reply src/main/kotlin/com/openlattice/launchpad/configuration/IntegrationTables.kt:24]: 100% did not know this was a thing but will definitely investigate for the future of queries

Comment (makosblade) [reply src/main/kotlin/com/openlattice/launchpad/configuration/IntegrationTables.kt:24]: Yep, gotta say I am a fan of multi-line strings now. Any changes will be in a separate PR b/c this is strictly a `merge develop to master` for release PR

**Pull request merged**